### PR TITLE
more flexible tactic uniqueness filter

### PR DESCRIPTION
**Pull request by mirefek**
`exclude_unique_tactic: false` ->`required_tactic_occurrence: 1`
`exclude_unique_tactic: true` ->`required_tactic_occurrence: 2`

tests pass

### CODE REVIEW

File: graph2tac/loader/data_server.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def __init__(self, data_dir : Path, dataset_config : DatasetConfig):
             file_data = self._data[name]
             self._load_file(file_data)
 
-        if self.dataset_config.exclude_unique_tactics:
+        if self.dataset_config.required_tactic_occurrence > 1:
```
Comment (LasseBlaauwbroek) [inline graph2tac/loader/data_server.py:393]: Maybe just move this check into `_exclude_unique_tactics`?

**Review by LasseBlaauwbroek — APPROVED** [review]
Looks good. Thanks. I'll let @jasonrute merge when he approves.

**Review by jasonrute — APPROVED** [review]
I'm approving since it isn't a big deal, and I'm fine with you merging as is, but I confused about having 1 be the lowest number, or am I mistaken and there is a difference between 0 and 1?  Is the idea that 0 means keep all tactics in the dataset even if never used, and 1 means take only tactics used at least once?  (And it just happens that our dataset has no tactics used zero times, because why would it?)

**Comment by mirefek** [comment]
There is not difference between 0 and 1.

**Comment by mirefek** [comment]
It was a bit easier to phrase the argument positively: "keep only tactics occuring at least k-times", and there is indeed no way to detect a tactic occuring 0 times.

**Comment by jasonrute** [comment]
Ok.  I'll merge it.

**Pull request merged**